client/web/src/enterprise/app/setup/steps/embeddings-step/AppEmbeddingsSetupStep.tsx

```diff
@@ -0,0 +1,149 @@
+import { FC, useContext, ChangeEvent, useState } from 'react'
+
+import { mdiGit } from '@mdi/js'
+import classNames from 'classnames'
+
+import { Button, H1, ScrollBox, Text, Icon, LoadingSpinner, Tooltip, Link, ErrorAlert } from '@sourcegraph/wildcard'
+
+import { LocalRepository } from '../../../../../graphql-operations'
+import { EnterprisePageRoutes } from '../../../../../routes.constants'
+import {
+    SetupStepsContext,
+    StepComponentProps,
+    useLocalRepositories,
+    useNewLocalRepositoriesPaths,
+} from '../../../../../setup-wizard/components'
+import { useScheduleRepoEmbeddingJobs } from '../../../../site-admin/cody/backend'
+import { AppNoItemsState } from '../../../components'
+
+import styles from './AppEmbeddingsSetupStep.module.scss'
+
+export const AppEmbeddingsSetupStep: FC<StepComponentProps> = ({ className }) => {
+    const { onNextStep } = useContext(SetupStepsContext)
+
+    const [scheduleRepoEmbeddingJobs] = useScheduleRepoEmbeddingJobs()
+    const [selectedRepository, setSelectedRepository] = useState<LocalRepository | null>(null)
+
+    const { paths, loading: pathsLoading, loaded: pathLoaded, error: pathsError } = useNewLocalRepositoriesPaths()
+
+    const {
+        repositories,
+        loading: repositoriesLoading,
+        loaded: repositoriesLoaded,
+        error: repositoriesError,
+    } = useLocalRepositories({ paths, skip: paths.length === 0 })
+
+    const handleNext = () => {
+        if (!selectedRepository) {
+            return
+        }
+
+        scheduleRepoEmbeddingJobs({ variables: { repoNames: [selectedRepository.name] } })
+        onNextStep()
+    }
+
+    const anyLoading = pathsLoading || repositoriesLoading
+    const anyError = pathsError || repositoriesError
+    const allLoaded = pathLoaded && repositoriesLoaded
+
+    return (
+        <div className={classNames(className, styles.root)}>
+            <div className={styles.description}>
+                <img
+                    src="https://storage.googleapis.com/sourcegraph-assets/cody-embeddings.png"
+                    alt=""
+                    className={styles.descriptionImage}
+                />
+
+                <H1 className={styles.descriptionHeading}>Level up your Cody(ing)</H1>
+
+                <Text className={styles.descriptionText}>What’s you preferred repository?</Text>
+
+                <Text className={classNames(styles.descriptionText, styles.descriptionTextSmall)}>
+                    Pick one repository to generate embeddings. This one-off choice supercharges a repo with even better
```

Comment:
```suggestion
                    Pick one repository to generate embeddings. This supercharges a repo with even better
```
I don't think we need the one-off part there the user can always pick more from settings.